src/collections/_documentation/workflow/integrations/integration-platform/webhooks-verifying-signature/node.md

```diff
@@ -0,0 +1,10 @@
+```javascript
+const crypto = require('crypto');
+
+function verifySignature(request, secret='') {
+  const hmac = crypto.createHmac('sha256', secret);
+  hmac.update(JSON.stringify(request.body), 'utf8');
+  const digest = hmac.digest('hex');
+  return digest === request.headers['sentry-hook-signature'];
```

Comment:
I think we need this to be capitalized to `'Sentry-Hook-Signature'`

Reply:
Hm in Express all the headers were coming back lower-case but yeah I'll change it :)

Reply:
@MeredithAnya whoops I lost track of this, could you check this latest update and approve / merge? :) thanks!